config/v1/types_cluster_version.go

```diff
@@ -215,6 +222,58 @@ type UpdateHistory struct {
 // ClusterID is string RFC4122 uuid.
 type ClusterID string
 
+// ClusterVersionCapabilitiesDefault is a constant representing the
+// default policy for optional, core cluster components.
+type ClusterVersionCapabilitiesDefault string
+
+const (
+	// IncludeClusterVersionCapabilities indicates that optional,
+	// core cluster components should be included by default.
+	IncludeClusterVersionCapabilities = "Include"
+
+	// ExcludeClusterVersionCapabilities indicates that optional,
+	// core cluster components should be excluded by default.
+	ExcludeClusterVersionCapabilities = "Exclude"
+)
+
+// ClusterVersionCapabilitiesSpec configures the installation of
```

Comment:
I think you need an example on this to explain the interaction of inclusionDefault and and these values.